relay-log/src/dashboard.rs

```diff
@@ -0,0 +1,43 @@
+use once_cell::sync::Lazy;
+use tokio::sync::broadcast;
+use tracing::Subscriber;
+use tracing_subscriber::Layer;
+
+/// Channel to deliver logs.
+pub static LOGS: Lazy<broadcast::Sender<Vec<u8>>> = Lazy::new(|| {
```

Comment:
_nit_: Could we make this private and instead expose an async fn / stream / function that returns a broadcast receiver?